app/src/main/java/org/openobservatory/ooniprobe/activity/MainActivity.java

```diff
@@ -66,6 +70,7 @@
     @Override
     protected void onCreate(Bundle savedInstanceState) {
         super.onCreate(savedInstanceState);
+
         setContentView(R.layout.activity_main);
         checkResources();
         TestData.getInstance(this, this).addObserver(this);
```

Comment:
My understanding of the [Activity lifecycle](https://developer.android.com/guide/components/activities/activity-lifecycle.html) is that `onResume` is called after `onCreate`; hence you don't need to `addObserver` this because you already do the same below inside of `onResume`.

Reply:
Didn't noticed it was duplicated